espresso-server/app/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
```

Comment:
wasn't this added in the other PR?

Reply:
Yeah, I'm going to fix that up.... I wanted to narrow down the problem, so I made a PR that only has the gradle changes.